clang/lib/3C/CheckedRegions.cpp

```diff
@@ -340,10 +345,11 @@ bool CheckedRegionFinder::isInStatementPosition(CallExpr *C) {
 }
 
 bool CheckedRegionFinder::isWild(CVarOption Cv) {
-  if (Cv.hasValue() &&
-      Cv.getValue().hasWild(Info.getConstraints().getVariables()))
-    return true;
-  return false;
+  const auto &E = Info.getConstraints().getVariables();
+  if (Cv.hasValue())
+    return Cv.getValue().hasWild(E) || Cv.getValue().hasParamWild(E);
+  else
+    return false;
```

Comment:
I'm wondering where the presence of type instantiations (i.e., that you return `malloc<int>(...)` and not `malloc(...)`) is showing up?

Reply:
Line 227 in the handling of call expressions